src/scheduler.cpp

```diff
@@ -367,13 +403,13 @@ void Scheduler::Worker::start() {
       thread = Thread(id, [=] {
         Thread::setName("Thread<%.2d>", int(id));
 
-        if (auto const& initFunc = scheduler->getThreadInitializer()) {
-          initFunc();
+        if (auto const& initFunc = scheduler->cfg.workerThread.initializer) {
+          initFunc(id);
```

Comment:
Was this 'id' argument missing? Does this change behaviour?

Reply:
Good spot. It's new with the `Config` interface.
See https://github.com/google/marl/issues/136#issuecomment-635261760 for context.
I've made the deprecated `setThreadInitializer` and `getThreadInitializer` functions wrap/unwrap the initializer so the signature is the same.